CONTRIBUTING.md

```diff
@@ -104,6 +104,15 @@ On Windows use the `Analyze Code Coverage for All Tests` on Visual Studio
 
 ![image](docs/vs-code-coverage.png)
 
+### Public Api
+
+Right now, we are using `Microsoft.CodeAnalysis.PublicApiAnalyzers` which will validate the public APi's. This analyzer will check if you changed a public property/method.
+
+If the solution is complaining about the publicApi, the easy way to solve is:
+
+1. Comment all target frameworks
```

Comment:
no need for this. Just use intellisense.

Reply:
just updated!